frontend/src/app/shared/components/storages/file-picker-modal/file-picker-modal.component.ts

```diff
@@ -75,6 +75,10 @@ export class FilePickerModalComponent extends FilePickerBaseModalComponent {
     return this.selection.size;
   }
 
+  public get showSelectAll():boolean {
+    return this.configuration.activeFeatureFlags.includes('storageFilePickingSelectAll');
```

Comment:
Note that this getter will fire on every change detection, which is more costly than having a field evaluated once (e.g., in ngOnInit) that resolves to a boolean field.

Reply:
True, changed to static property filled in constructor.

wanted to rebase to test it with current dev, so no incremental diff.